schema_utils.py

```diff
@@ -229,12 +229,12 @@ def sanitize_url(obj):
         """Takes a string representing a URL and sanitizes it.
 
         Args:
-            obj: a string representing a URL.
+            obj: str. a string representing a URL.
 
         Returns:
-            An empty string if the URL does not start with http:// or https://
-            except when the string is empty. Otherwise, returns the original
-            URL.
+            url(str). An empty string if the URL does not start with http:// or
```

Comment:
Just `str` (for the typeinfo). url(str) isn't a valid type.

Reply:
Done.